packages/ui/src/components/GalleryMedia/GalleryMedia.tsx

```diff
@@ -0,0 +1,76 @@
+import { ArrowRightIcon, PhotoIcon, VideoCameraIcon } from '@heroicons/react/24/outline';
+import type { Culture, NewsroomGallery } from '@prezly/sdk';
+import { getGalleryThumbnail } from '@prezly/theme-kit-core';
+import { UploadcareImage } from '@prezly/uploadcare-image';
+import { twMerge } from 'tailwind-merge';
+
+import { Link } from '../Link';
+
+export interface Props {
+    className?: string;
+    gallery: NewsroomGallery;
+    locale?: Culture['code'];
+}
+
+export function GalleryMedia({ className, gallery, locale }: Props) {
+    const { name, images_number, videos_number, uuid } = gallery;
+    const cover = getGalleryThumbnail(gallery);
+
+    return (
+        <Link
+            href={`/media/album/${uuid}`}
+            className={twMerge('relative w-full md:w-max group', className)}
+            contentClassName="p-0"
+            localeCode={locale}
+        >
+            {cover && (
+                <UploadcareImage
+                    className={twMerge(
+                        'aspect-[17/12] md:aspect-[6/7] w-full max-w-full md:max-w-[258px]',
```

Comment:
258px sounds a bit too specific. Wouldn't 256px work here, so that we can use `md:max-w-64` class name and adhere to the grid?

Reply:
Sure will update, used the width from the design system that's why I ended with this

Reply:
Had to use `md:max-w-[256px]` here as max-width does not use the usual size variations of tailwind